src/VGriveClient.vala

```diff
@@ -228,26 +228,41 @@ namespace App {
                 return false;
             }
         }
-        private void check_deleted_files () {
+        
+         private void check_deleted_files () {
```

Comment:
Be carefull with identation, now there is one extra space

Reply:
Sorry! Did not see that :sweat_smile: